test/domain/relation/establish.spec.ts

```diff
@@ -10,6 +10,12 @@ describe('domain/relation/establish', () =>
         const relation = await database.findRecord(RECORD_TYPE_RELATION, QUERY_EXISTING_RELATION);
 
         expect(relation?.id).toBeDefined();
+
+        const creator0 = await database.readRecord(RECORD_TYPE_CREATOR, REQUESTER2.id);
```

Comment:
REQUESTOR2.id is the `id` of creator 1, but the value is stored in creator0 variable. That is confusing.